test/performance/range/views/view_all_benchmark.cpp

```diff
@@ -7,12 +7,13 @@
 
 #include <deque>
 #include <list>
+#include <seqan3/std/ranges>
 #include <string>
 #include <vector>
 
 #include <benchmark/benchmark.h>
 
-#include <seqan3/std/ranges>
+#include <range/v3/view/all.hpp>
```

Comment:
But what is used from the ranges library from all.hpp that is not provided by the standard ranges header?

Reply:
The `::ranges::view::all` that is benchmarked.
My first idea was to excluded the benchmark if there is no range-v3, but Marcel disagreed.

We also still have range-v3 includes in many files....

Reply:
Ok, I am confused now. Should it then not test the standard views::all? Why are we interested in the ranges views::all if we don't use it? I wouldn't disable it. I would just test the views::all that we get. In a life without range-v3 this becomes useless anyway.

Reply:
Yes, this was my other idea, too.
If cpp17 -> useless because they are the same (`std::views::all` = `::ranges::views:all`)
If cpp20 -> useless because why would you use the range-v3 version?

I can change it, I just didn't because Marcel had objections.